rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -225,4 +228,48 @@ public static boolean isValidJSONObject(String str) {
             return false;
         }
     }
+
+    public static void verifyUserAuthorization() {
+        UserVerifyAuthTask.getInstance().run();
+    }
+
+    private static class UserVerifyAuthTask {
```

Comment:
Rename to `UserAuthorizationVerificationTask`

Reply:
done